test/expect.js

```diff
@@ -397,6 +397,35 @@ describe('expect', function () {
     err(function(){
       expect({foo: 'bar'}).to.be.empty;
     }, "expected { foo: \'bar\' } to be empty");
+
+    err(function(){
+      expect(0).to.be.empty;
+    }, "expected 0 to be empty");
```

Comment:
FYI the current behaviour is `expect(0).to.be.empty` passes. Can you please explain why you want it to now fail?